resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/features/BasicExecutorFeatureStep.scala

```diff
@@ -172,7 +172,8 @@ private[spark] class BasicExecutorFeatureStep(
         .addToImagePullSecrets(kubernetesConf.imagePullSecrets(): _*)
         .endSpec()
       .build()
-    SparkPod(executorPod, containerWithLimitCores)
+
+    SparkPod(executorPod, executorContainer)
```

Comment:
Why this change?

Reply:
I think it was me resolving merge incorrectly, will fix